hazelcast/proxy/topic.py

```diff
@@ -70,6 +71,22 @@ def publish(self, message: MessageType) -> Future[None]:
         request = topic_publish_codec.encode_request(self.name, message_data)
         return self._invoke(request)
 
+    def publish_all(self, messages: typing.Sequence[MessageType]) -> Future[None]:
+        """Publishes a list of messages to all subscribers of this topic
+
+        Args:
+            messages: The message list to be published
```

Comment:
It doesn't have to be a list, maybe just say `Messages to be published`

Reply:
Fixed at https://github.com/hazelcast/hazelcast-python-client/pull/570/commits/d2d7dc0387ac8e704d4f74533cc1f9d24854773e.